src/digitalmarketplace/components/cookie-banner/template.njk

```diff
@@ -3,7 +3,7 @@
 {% set cookieInfoUrl = params.cookieInfoUrl if params.cookieInfoUrl else '/cookies' %}
 {% set cookieSettingsUrl = params.cookieSettingsUrl if params.cookieSettingsUrl else '/users/cookie-settings' %}
 
-<div id="{{ componentID }}" class="dm-cookie-banner govuk-width-container" data-module="dm-cookie-banner" role="region" aria-describedby="dm-cookie-banner__heading">
+<div id="{{ componentID }}" class="dm-cookie-banner govuk-width-container" data-module="dm-cookie-banner" role="region" aria-labelledby="dm-cookie-banner__heading">
```

Comment:
Ah, interesting! I wonder if we should let Notify know (this code was copied from [their banner](https://github.com/alphagov/notifications-admin/blob/182c47dfa6af435b5b2a8f409a95bc8825b85f2a/app/templates/components/cookie-banner.html)).

Reply:
Which was in turn [adapted from GOVUK Publishing Components](https://github.com/alphagov/notifications-admin/commit/3a6537f49f2f682c3b8f981234e491d2cac47715), which now uses an `aria-label` property set to "cookie banner".

We could do the same? There should only be one instance of this component per page anyway.